managed/services/management/dbaas/mock_dbaas_client_test.go

```diff
@@ -45,29 +45,6 @@ func (_m *mockDbaasClient) ApproveInstallPlan(ctx context.Context, in *controlle
 	return r0, r1
 }
 
-// CheckKubernetesClusterConnection provides a mock function with given fields: ctx, kubeConfig
```

Comment:
This mock update was skipped by CI because `make gen` invoked only in API workflow.